src/rtp/ext.rs

```diff
@@ -188,6 +188,25 @@ impl Extension {
         "unknown"
     }
 
+    pub(crate) fn is_serialized(&self) -> bool {
+        if let Self::UnknownUri(_, s) = self {
+            // Check if this Arc contains the SdpUnknownUri.
+            let is_sdp = (s as &(dyn Any + 'static))
+                .downcast_ref::<SdpUnknownUri>()
+                .is_some();
+
+            // If it is the SdpUnknownUri, we are not serializing. If this happens,
+            // it's probably a bug. The only way to construct SdpUnknownUri is via SDP,
+            // but those values are only for Eq-comparison vs the values in Session.
+            // The SdpUnknownUri should not even try to be serialized.
+            // TODO: Should we panic instead?
```

Comment:
Is it possible to declare an extension in SDP, but not register a serializer?

Reply:
The extensions are negotiated, so both sides must have them.

If the remote peer sends an extension we don't recognize, we will just drop it. No serializer needed.

Reply:
Exactly, so panic?

Reply:
Alright. Panic :)